pkg/translator/azure/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers, atoulme]
```

Comment:
If you want to add yourself there, feel free

Reply:
Okay, I can do that.